assets/xml/objects/object_gi_seahorse.xml

```diff
@@ -1,12 +1,13 @@
 ﻿<Root>
     <File Name="object_gi_seahorse" Segment="6">
-        <!-- <Blob Name="object_gi_seahorse_Blob_000A50" Size="0x40" Offset="0xA50" /> -->
-        <DList Name="object_gi_seahorse_DL_000A90" Offset="0xA90" />
-        <DList Name="object_gi_seahorse_DL_000B20" Offset="0xB20" />
-        <DList Name="object_gi_seahorse_DL_000F08" Offset="0xF08" />
-        <Texture Name="object_gi_seahorse_Tex_000F10" OutName="tex_000F10" Format="ia8" Width="16" Height="8" Offset="0xF10" />
-        <Texture Name="object_gi_seahorse_Tex_000F90" OutName="tex_000F90" Format="i4" Width="16" Height="8" Offset="0xF90" />
-        <Texture Name="object_gi_seahorse_Tex_000FD0" OutName="tex_000FD0" Format="i8" Width="32" Height="32" Offset="0xFD0" />
-        <Texture Name="object_gi_seahorse_Tex_0013D0" OutName="tex_0013D0" Format="ia8" Width="32" Height="32" Offset="0x13D0" />
+    <!-- Seahorse model not inside a bottle -->
```

Comment:
Same thing I mentioned for the loach applies here; might be better to put this in the comment above the <File> node.